resource/certificate.go

```diff
@@ -33,6 +34,11 @@ func CreateCertificate(org string, keyPath string, certPath string) error {
 	common.Configuration.ServerCertificate = path.Join(certPath, config.HZN_FSS_CERT_FILE)
 	common.Configuration.ServerKey = path.Join(keyPath, config.HZN_FSS_CERT_KEY_FILE)
 
+	if fileExists(common.Configuration.ServerCertificate) && fileExists(common.Configuration.ServerKey) {
+		glog.V(5).Infof(reslog(fmt.Sprintf("ESS self signed cert and key already exist in %v, %v, skip creating...", common.Configuration.ServerCertificate, common.Configuration.ServerKey)))
```

Comment:
tested this and it worked!
Only comment I have is whether we should make this log level 3 instead of 5 so we can see it by default. Seems like this is pretty important... thoughts?